packages/now-build-utils/test/unit.builds-and-routes-detector.test.ts

```diff
@@ -1105,7 +1145,19 @@ it('Test `detectRoutes` with `featHandleMiss=true`', async () => {
       builders!,
       featHandleMiss
     );
-    expect(defaultRoutes).toStrictEqual(expectedRoutes);
+    expect(defaultRoutes).toStrictEqual([
+      { handle: 'miss' },
+      {
+        src: '^/api/([^\\/]+)$',
+        dest: '/api/[endpoint].js?endpoint=$1',
```

Comment:
This shouldn't be using the extension

Reply:
At this time, I'm not renaming output files so the routes will need to remain here.

Reply:
Thanks, I updated the PR to remove the file extension 👍